README.md

```diff
@@ -140,23 +140,77 @@ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/privat
 
 *Important*: Make sure that the certificate files are named exactly `ssl-certificate.key` and `ssl-certificate.crt`. If you have procured the certificate from a different source, rename them to the prescribed filenames, and ensure that they are located in the same folder as the `install.sh` script. They will get uploaded to the Chef Habitat supervisor during the install.
 
+### Airgapped Installation (Recommended prep work, if airgapped)
+
+In order to install the on-prem Chef Habitat Builder in an airgapped (no direct Internet access) environment, the following preparatory steps are helpful:
+
+1. Download the [Zip archive](https://github.com/habitat-sh/on-prem-builder/archive/master.zip) of the on-prem-builder repo
+
+    ```bash
+    curl -LO https://github.com/habitat-sh/on-prem-builder/archive/master.zip
+    ```
+
+1. Download the Chef Habitat [cli tool](https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-%24latest-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux)
+
+    ```bash
+    curl -Lo hab.tar.gz https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-%24latest-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux
+    ```
+
+1. Create the Habitat Builder starter kit bundle and download it
+
+     ```bash
+     git clone https://github.com/habitat-sh/on-prem-builder.git
+     export DOWNLOAD_DIR=/some/base/download/directory
+     cd on-prem-builder
+     hab pkg download --target x86_64-linux --channel stable --file quickstart_lists/builder_x86_64-linux_stable --download-directory ${DOWNLOAD_DIR}/builder_starter_kit
+     ```
+
+1. Create any additional starter kit Builder bootstrap bundles as documented in the [Bootstrap Builder](https://github.com/habitat-sh/on-prem-builder/tree/master#bootstrap-builder-with-habitat-packages) section of this README. You can specify `--download-directory ${DOWNLOAD_DIR}/builder_bootstrap` argument to the download command in order to consolidate all bootstrap packages in a single directory
+1. Zip up all the above, transfer and unzip on the Linux system where Builder will be deployed
+1. From the zip archive, install the `hab` binary somewhere in $PATH and ensure it has execute permissions:
```

Comment:
Maybe separate this part into its own subsection to differentiate the work that has to be done in airgap prep, vs on the airgap environment

Reply:
Addressed in https://github.com/habitat-sh/on-prem-builder/pull/212/commits/38454a460a0f673bf5f0cde08392dfcb3e36b1f6